Sources/CassandraClient/Configuration.swift

```diff
@@ -51,6 +51,48 @@ extension CassandraClient {
         /// Sets the cluster's consistency level. Default is `.localOne`.
         public var consistency: CassandraClient.Consistency?
 
+        /// The load balancing strategy to use. Default is `nil` which uses ``LoadBalancingStrategy/dataCenterAware(_:)``.
+        public var loadBalancingStrategy: LoadBalancingStrategy?
+
+        /// A struct representing the load balancing strategy.
+        public struct LoadBalancingStrategy: Hashable {
+            enum Backing: Hashable {
+                case roundRobin(RoundRobin)
+                case dataCenterAware(DataCenterAware)
+            }
+            public struct RoundRobin: Hashable {
+                public init() {}
+            }
+            public struct DataCenterAware: Hashable {
+                /// Sets the local data center name for DC-aware routing policy.
+                /// When set, a DC-aware load balancing policy will be used that prioritizes hosts from this data center.
+                public var localDataCenter: String?
+
+                /// Creates a new data center aware load balancing strategy.
+                ///
+                /// - Parameters:
+                ///   - localDataCenter: Sets the local data center name for DC-aware routing policy.
+                public init(
+                    localDataCenter: String? = nil
+                ) {
+                    self.localDataCenter = localDataCenter
+                }
+            }
+
+            var backing: Backing
+
+            /// Returns a new round robin load balancing strategy.
+            public static func roundRobin(_ roundRobin: RoundRobin) -> Self {
+                .init(backing: .roundRobin(roundRobin))
+            }
+
```

Comment:
```suggestion

            public static var roundRobin: Self { .roundRobin(.init()) }

```

For convenience, since there is no additional round robin config.